contents/docs/libraries/ios/configuration.mdx

```diff
@@ -0,0 +1,388 @@
+---
+title: iOS SDK configuration
+sidebarTitle: iOS
+sidebar: Docs
+showTitle: true
+github: 'https://github.com/PostHog/posthog-ios'
+icon: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/docs/integrate/ios.svg
+features:
+  eventCapture: true
+  userIdentification: true
+  autoCapture: true
+  sessionRecording: true
+  featureFlags: true
+  groupAnalytics: true
+  surveys: true
+  llmAnalytics: false
+---
+## Autocapture configuration
+
+You can enable or disable autocapture through the `PostHogConfig` object.
+
+<AutocaptureConfig />
+
+## Flush configuration
+
+You can set the number of events in the configuration that should queue before flushing.
+Setting this to `1` will send events immediately and will use more battery. This is set to `20` by default.
+
+```swift
+configuration.flushAt = 1
+```
+
+You can also manually flush the queue:
+
+```swift
+PostHogSDK.shared.capture("logged_out")
+PostHogSDK.shared.flush()
+```
+
+## Offline behavior
+
+The PostHog iOS SDK will continue to capture events when the device is offline. The events are stored in a queue in the device's file storage and are flushed when the device is online.
+
+- The queue has a maximum size defined by `maxQueueSize` in the configuration.
+- When the queue is full, the oldest event is deleted first.
+- The queue is flushed only when the device is online.
```

Comment:
This might be good for installation? I don't necessarily think this is a configuration point, we could link out on the `maxQueueSize` spot.